### PR TITLE
Make Benefits consistently a top-level section with h2 in source files

### LINKED ISSUE #3924: Benefits sometimes nested under Intent in source files (including template)
The Benefits section in Understanding documents is always displayed as a level-2 heading and included in the table of contents, after Intent.

However, a handful of files - _including `understanding-template.html`_ - instead nest Benefits within Intent, which the build process corrects, pulling it back out to a top-level section with `h2`.

I'm not sure if there's history behind this discrepancy, but it seems like we ought to make the source files consistent - and at the very least, make the template reflect reality.

A search for `<h3>Benefits` should find the source files in question (as opposed to `<h2>Benefits` for ones that fit the current build behavior).

### PR DESCRIPTION
**Pull request by kfranqueiro**
Fixes #3924.

Also addresses the Label in Name instance in #3922.

The diff will be much easier to compare with whitespace ignored (`?w=1`).

This amends all instances of the Benefits section in Understanding pages to be a top-level section with a level-2 heading.

I've tested this with both the XSLT and Eleventy build processes to confirm this doesn't actually change any content, since both build processes already ensure Benefits is a top-level section with level-2 heading. The one exception is the removal of the redundant first Benefits section in Understanding 2.5.3 Label in Name, which only meaningfully affects the XSLT process, as the Eleventy process already worked around that issue.

Note: I don't know what `identify-changes.html` is supposed to be, but I updated it regardless; it seems to only contain template boilerplate and is not linked from anywhere.

### CODE REVIEW

File: understanding/21/animation-from-interactions.html [hunk 1]
```diff
@@ -27,21 +27,20 @@ <h2>Intent</h2>
...
-                <li>Persona Quote: "Stop that extra movement! You are making me so dizzy I cannot concentrate. Now I have to turn off my computer and go lie down."</li>
-            </ul>
-          </li>
-        </ul>
-			</section>
+			<p><strong>How can a website reduce the chances of triggering a vestibular disorder?</strong> Choose any one of the following solutions. Avoid using unnecessary animation. Provide a control for users to turn off non-essential animations from user interaction. Take advantage of the reduce motion feature in the user agent or operating system.</p>
+
+			<p><strong>What about movement caused by a user scrolling a page?</strong> Moving new content into the viewport is essential for scrolling. The user controls the essential scrolling movement so it is allowed. Only add non-essential animation to the scrolling interaction in a responsible way. Always give users the ability to turn off unnecessary movement.</p>
+		</section>
+		<section id="benefits">
```
Comment (Ilovemesomehim1) [inline understanding/21/animation-from-interactions.html:34]:
```suggestion
		<section id="benefits">```suggestion
		<section id="benefits">```suggestion
		<section id="benefits">********
```

```

```

**Review by Ilovemesomehim1 — APPROVED** [review]

**Comment by patrickhlauke** [comment]
can we limit reviews/approvals to w3c member users? tired of these random muppet accounts (that clearly seem to somehow game github for some reason) making BS comments/suggestions.

**Comment by bruce-usab** [comment]
> can we limit reviews/approvals to w3c member users?

Yes., Ken just now changed the setting after checking in with Kevin and Mike and me.

**Review by patrickhlauke — APPROVED** [review]

**Comment by kfranqueiro** [comment]
@mbgower did you have to manually deal with merge conflicts? I am concerned that the changeset I see actually merged may have changed more than this PR originally did...

**Comment by kfranqueiro** [comment]
I confirmed breakage in the merge; followed up in #4042.

**Pull request merged**